### PR TITLE
#166374546 Add lendingHistory model

### PR DESCRIPTION
**Pull request by Adekoreday**
#### What does this PR do?

create lendingHistory model,
add lendingHistory migrations,

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
```
clone or download repository
npm  install
add .env file
include PORT and database parameters in the .env file.
npm run db:roll


```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#166374546

### CODE REVIEW

**Review by KvNGCzA — CHANGES_REQUESTED** [review]
Rebase your branch
done

File: server/database/models/users.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'userId',
       onDelete: 'CASCADE'
     });
+
+      Users.hasMany(models.lendingHistory, {
```
Comment (KvNGCzA) [inline server/database/models/users.js]: When you rebase, please ensure to correct this
thanks done

File: server/database/migrations/20190531184608-create-lending-history.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
```
Comment (KvNGCzA) [inline server/database/migrations/20190531184608-create-lending-history.js]: Shouldn't this table also have bookId, date it was borrowed and date it was be returned?

File: server/database/models/lendinghistory.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+/* eslint-disable */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const lendingHistory = sequelize.define('lendingHistory', {
+    status: DataTypes.STRING,
+    duration: DataTypes.STRING
```
Comment (allebd) [inline server/database/models/lendinghistory.js]: You did not define userId and bookId
They are foreign keys which are defined in the association

File: server/database/migrations/20190531184608-create-lending-history.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('lendingHistories', {
```
Comment (KvNGCzA) [inline server/database/migrations/20190531184608-create-lending-history.js]: Why is your table named in Camel Case?

File: server/database/migrations/20190531184608-create-lending-history.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      status: {
+        type: Sequelize.STRING
+      },
+         duration: {
```
Comment (KvNGCzA) [inline server/database/migrations/20190531184608-create-lending-history.js]: fix your indentation

**Review by KvNGCzA — CHANGES_REQUESTED** [review]

File: server/database/migrations/20190531184608-create-lending-history.js (hunk 4 above)
Comment (Adekoreday) [reply server/database/migrations/20190531184608-create-lending-history.js]: will reflect on this just seeing it

File: server/database/migrations/20190531184608-create-lending-history.js (hunk 5 above)
Comment (Adekoreday) [reply server/database/migrations/20190531184608-create-lending-history.js]: thanks will reflect on this

**Review by Lundii — DISMISSED** [review]
Good

File: server/database/models/book.js [hunk 6]
```diff
@@ -15,7 +15,12 @@ export default (sequelize, DataTypes) => {
       onDelete: 'CASCADE'
     });
 
-    Book.hasMany(models.Fine, {
+   Book.hasMany(models.Fine, {
```
Comment (allebd) [inline server/database/models/book.js]: Kindly work on your indentation

File: server/database/models/book.js [hunk 7]
```diff
@@ -15,7 +15,12 @@ export default (sequelize, DataTypes) => {
...
+      foreignKey: 'bookId',
+      onDelete: 'CASCADE'
+    })
+
+  Book.hasMany(models.Lendinghistory, {
```
Comment (allebd) [inline server/database/models/book.js]: Kindly work on this indentation also

Comment (KvNGCzA) [inline server/database/models/book.js]: `LendingHistory`

**Review by KvNGCzA — CHANGES_REQUESTED** [review]

File: server/database/models/book.js (hunk 7 above)
Comment (Adekoreday) [reply server/database/models/book.js]: thanks will do

Comment (Adekoreday) [reply server/database/models/book.js]: ok will change it

**Review by KvNGCzA — APPROVED** [review]

**Pull request merged**